File: Programming Fundamentals/Labs And Homeworks/BasicSyntaxExercise/Ages.java
package basicSyntaxExercise;

import java.util.Scanner;

public class Ages {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int age = Integer.parseInt(scanner.nextLine());
        if (age >= 0 && age <= 2) {
            System.out.println("baby");
        } else if (age >= 3 && age <= 13) {
            System.out.println("child");
        } else if (age >= 14 && age <= 19) {
            System.out.println("teenager");
        } else if (age >= 20 && age <= 65) {
            System.out.println("adult");
        } else if (age >= 66) {
            System.out.println("elder");
        }
    }
}

File: JS Front-End/JS Syntax Fundamentals - Exercise/ages.js
function ages(age) {
    if (age >= 0 && age <= 2) {
        console.log('baby');
    }else if (age >= 3 && age <= 13) {
        console.log('child');
    }else if (age >= 14 && age <= 19) {
        console.log('teenager');
    }else if (age >= 20 && age <= 65) {
        console.log('adult');
    }else if (age >= 66) {
        console.log('elder');
    }else {
        console.log('out of bounds');
    }

}
ages(20);
ages(100);
ages(-1);